bookmarks.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Quiz</title>
+  </head>
+  <body>
+    <h1>Quiz App</h1>
+    <main>
+      <section class="index-card">
+        <h2>Question</h2>
+        <p class="bookmark">
+          <img src="./images/Icon-Bookmark-Set.svg" alt="Icon: Bookmark" />
+        </p>
+        <p>
+          Tootsie roll pie topping powder sugar plum souffl&eacute;. Gingerbread
+          sugar plum tiramisu powder macaroon candy canes drag&eacute;e icing.
+          Sweet macaroon tootsie roll chocolate chocolate cake. Tootsie roll
+          muffin donut apple pie gummies powder. Lollipop candy canes bonbon
+          sesame snaps danish brownie croissant tiramisu. Oat cake pastry
+          pudding ice cream fruitcake. Muffin pudding croissant pudding tart oat
+          cake caramels sugar plum icing.
+        </p>
+        <ul class="categories">
+          <li>#categorie</li>
+          <li>#categorie</li>
+          <li>#categorie</li>
+        </ul>
+        <button>Antwort --></button>
+      </section>
+      <section class="index-card">
+        <h2>Question</h2>
+        <p class="bookmark">
+          <img src="./images/Icon-Bookmark-Set.svg" alt="Icon:Set Bookmark" />
+        </p>
+        <p>
+          Cupcake ipsum dolor sit amet cake croissant fruitcake muffin. Sweet
+          bonbon candy powder cheesecake muffin sesame snaps jujubes. Muffin
+          macaroon cheesecake cookie souffl&eacute; cookie cheesecake lollipop
+          candy canes. Halvah cotton candy sesame snaps chocolate bar. Sugar
+          plum dessert drag&eacute;e macaroon chupa chups. Cake cheesecake wafer
+          cotton candy. Danish jelly halvah jelly beans tootsie roll donut.
+          Halvah cheesecake halvah jujubes sweet roll biscuit croissant wafer.
+          Cake halvah chocolate bar sugar plum apple pie.
+        </p>
+        <ul class="categories">
+          <li>#categorie</li>
+          <li>#categorie</li>
+          <li>#categorie</li>
+        </ul>
+
+        <button>Antwort --></button>
```

Comment:
schöne Idee mit dem Pfeil! schau mal hier in der Liste von Julia (https://www.vioma.de/de/wiki/tools/html-sonderzeichen/#HTML%20Sonderzeichen%20-%20Favoriten) da gibts noch ganz viele tolle Pfeile als html-code :)(auch Zeile 32)